man/fragments/ex-navs_tab.Rmd

```diff
@@ -0,0 +1,223 @@
+```{r setup, include = FALSE}
```

Comment:
The real driver is roxygen2's `@includeRmd` tag that you can use to inline Rmd into documentation. That's super helpful for long docs, and it's basically a child Rmd for your docs.

Storing them `man/fragments` is [my personal preference](https://www.garrickadenbuie.com/blog/dry-vignette-and-readme/) since it feels similar to `man/figures`, but technically we could put these bits anywhere.